### DISCUSSION TITLE
ServiceWorker together with Firefly in subdirectory

### DISCUSSION THREAD

**Discussion by Kaemmelot** [firefly-iii/firefly-iii#6324] [Firefly III]
Hi,

I'm trying to find out whether I misconfigured something.
Firefly runs in a subdirectory of my domain via apache2. I redirected the traffic by using `Alias /firefly/ /var/www/firefly/public/`. My browser console (Chrome 104.0.5112.80) shows these messages:
![firefly-error1](https://user-images.githubusercontent.com/5353825/183311188-60de1c5d-f182-46b6-9d05-e2cb3692912c.jpg)

But even when I allow the service worker by setting `Header set Service-Worker-Allowed "/"` in apache, the service worker still does not function:
![firefly-error2](https://user-images.githubusercontent.com/5353825/183311240-b39e6dca-ce25-4c48-9e5b-6d4b51a77bf1.jpg)

Is there something I missed?

**Comment by JC5** [comment]
Not sure. But the service worker is going to be removed in future versions. If the rest works, you can ignore it.

**Reply by Kaemmelot** [reply]
Thank you for the fast reply and the effort you put in this project.
